website/static/css/site.css

```diff
@@ -341,11 +345,17 @@ li.pointer {
 .comment-container {
     margin-top: 10px;
     margin-bottom: 10px;
+    border-left-style: dashed;
+    border-left-color: lightgrey;
```

Comment:
Related to a previous comment: is it possible to use any Bootstrap helper classes (e.g. `text-muted` instead of this custom CSS)?